YIPS/yip-13.md

```diff
@@ -0,0 +1,28 @@
+---
+yip: 13
+title: Remove YFI burning
+status: WIP
+author: Sunil Srivatsa (@alphastorm)
+discussions-to: <TODO>
+created: 2020-07-24
+---
+
+## Simple Summary
+Remove YFI burning from the protocol.
+
+## Abstract
+All yEarn fees (currently ~$60/wk) are routed to this Vault and normalized to aDAI.
```

Comment:
- Which "this Vault"?
- Aren't they normalized to yCRV?

Reply:
Generalized the language to just yEarn protocol fees.